src/style.css

```diff
@@ -0,0 +1,31 @@
+@import "./styles/card.css";
+@import "./styles/stars.css";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
don't drop style's like margin and padding with "ALL TAG = *", for border-box it's okay